README.md

```diff
@@ -21,32 +19,66 @@
-* I didn't go through with the **Final Test Target Configurations** with regards to the [Kif Cocoapod](https://github.com/kif-framework/KIF#final-test-target-configurations)
+The view controller also has a couple more instance variables. `simonSaysGame` is a reference to the game engine, and `buttonsClicked` records the number of buttons that have been clicked so far, which you may find to be helpful in implementing this application.
 
-```swift
-Final Test Target Configurations
+## View
 
-You need your tests to run hosted in your application. Xcode does this for you by default when creating a new testing bundle target, but if you're migrating an older bundle, follow the steps below.
+The view in `Main.storyboard` has also been partially set up for you. You'll see there is a **Start Game** button. It is already connected to the `startGameTapped(_:)` IB action in the view controller. The label indicating whether the user has one or lost has been created for you, too, and connected as an IB outlet.
 
-First add your application by selecting "Build Phases", expanding the "Target Dependencies" section, clicking on the "+" button, and in the new sheet that appears selecting your application target and clicking "Add".
+## Your Tasks
 
-Next, configure your bundle loader. In "Build Settings", expand "Linking" and edit "Bundle Loader" to be $(BUILT_PRODUCTS_DIR)/MyApplication.app/MyApplication where MyApplication is the name of your app. Expand the "Testing" section and edit "Test Host" to be $(BUNDLE_LOADER). Also make sure that "Wrapper Extension" is set to "xctest".
+There's still a lot of work to be done, though. Here's what you need to do to finish this lab:
 
-The last step is to configure your unit tests to run when you trigger a test (⌘U). Click on your scheme name and select "Edit Scheme…". Click on "Test" in the sidebar followed by the "+" in the bottom left corner. Select your testing target and click "OK".
-```
+1. Create a colored button in each corner of the UI.
+	* These buttons should be roughly 100 pixels by 100 pixels square. Remember: You can drag their size handles to change their size, or change them in the Attributes Inspector.
+	* They have no text, so you should remove the default text.
+	* Finally, you should change their background colors. This can also be done in the Attributes Inspector.
+2. Create IB actions for each of the colored buttons in the view controller, and wire them up to these actions.
+3. Hide the `winLabel` when the view initially loads. It shouldn't be shown until the game is over.
+4. When a button is pressed, check to see if the user has won the game. Remember, `simonSaysGame` has a `gameWon()` method that will return `true` if the user won the game.
+5. Once you can determine if the user has won or lost, change the text in the `winLabel` to indicate if the user won or lost.
```

Comment:
Regarding #4. I think we can change the wording here a little to help the student out. Instead of just stating

> When a button is pressed, check to see if the user has won the game.

I would like to add that they should first call on the guessYellow, red, ... methods available to them with the `simonSaysGame` Something like this (although I'm not too fond of how I'm wording it):

Implement each method that represents the action when a color button is tapped. If the yellow button is tapped, make sure to first call on `simonSaysGame.guessYellow()` within that method.